ftc.py

```diff
@@ -87,7 +103,10 @@ def initialize_transcripts():
 
             transcripts.append((source, episode, idx, text, max_logprob)) 
 
+             if source in DESIRED_SOURCES:
+                filtered_indices.append(len(transcripts) - 1)
     transcripts.sort(key=lambda e: e[4])
+    filtered_transcripts = [transcripts[i] for i in filtered_indices]
```

Comment:
@yanirmr Would'n the indices be wrong after the "logprob sort" is applied one row above?